cuml/bench.py

```diff
@@ -8,6 +8,18 @@
 import sklearn
```

Comment:
do we really need to copy this file to cuml/sklearn/xgboost - given that it is the same everywhere?

Reply:
Not in scope of this PR